tests/base/boring_model.py

```diff
@@ -117,15 +121,16 @@ def test_epoch_end(self, outputs) -> None:
         torch.stack([x["y"] for x in outputs]).mean()
 
     def configure_optimizers(self):
-        optimizer = torch.optim.SGD(self.layer.parameters(), lr=0.1)
+        optimizer_class = getattr(torch.optim, self.optimizer_name)
+        optimizer = optimizer_class(self.parameters(), lr=self.learning_rate)
         lr_scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=1)
         return [optimizer], [lr_scheduler]
 
     def train_dataloader(self):
-        return torch.utils.data.DataLoader(RandomDataset(32, 64))
+        return torch.utils.data.DataLoader(RandomDataset(32, 64), batch_size=self.batch_size)
```

Comment:
Can you use in_features there too. If I change it, it will break

Reply:
Thanks @tchaton! I totally agree! Done the requested change :+1: